starter-code/css/main.css

```diff
@@ -1,8 +1,168 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+
+html, body {
+    margin:0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Varela Round', sans-serif;
+    /*line-height: 3.5em;*/
+    color:#1A1A1A;
+}
+
+.nav-bar {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color:#FFF;
+    width: 100%;
+
+}
+
+.nav-bar ul{
+    width: 40%;
+    margin-right: 20px;
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+}
+
+.nav-bar ul li{
+    padding: 2px;
+    font-size: 1em;
+}
+
+.nav-bar div img {
+    width: 125px;
+}
+
+.conteiner {
+    background-image:url("./../images/landing.jpg");
```

Comment:
You could add the property: 
background-position: center;
so the image would be centered and not only the top part of the image would appear due to the fact that you fixed a height to 650px and that the image is a bigger than that.